_posts/2020-11-26-torch-brain-segmentation/torch_brain_segmentation.Rmd

```diff
@@ -0,0 +1,705 @@
+---
+title: "Brain image segmentation with torch"
+description: > 
+  The need to segment images arises in various sciences and their applications, many of whom are vital to human (and animal) life. In this introductory post, we train a U-Net to mark lesioned regions on MRI brain scans.
+
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchtabular
+date: 11-26-2020
+categories:
+  - Torch
+  - R
+  - Image Recognition & Image Processing
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/scans.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+## When *what* is not enough
+
+True, sometimes it's vital to distinguish between different kinds of objects. Is that a car speeding towards me, in which case I'd better jump out of the way? Or is it a huge terrier (in which case I'd probably do the same)? Often in real life though, instead of coarse-grained *classification*, what is needed is fine-grained *segmentation*.
```

Comment:
It feels weird to me to start with True, here. I think you can just cut it. I started looking around for a question or a something I had missed, since it somehow implies that it's in response to something.

Reply:
Do you mean terrier like the dog? If so, maybe just say dog—terriers are pretty little (generally speaking—pit bulls are technically American Staffordshire Terriers, but few people actually know that).

Reply:
(((like we said on slack)))